.gitignore

```diff
@@ -7,3 +7,17 @@ maven-deps/
 .DS_Store
 archived/
 cygnus-ngsi/ignore
+.metadata/*
+.recommenders/*
+telefonica_checkstyle.xml
+sun_checkstyle.xml
+telefonica_checkstyle.xml
+RemoteSystemsTempFiles/*
+*.prefs
+cygnus-common/.classpath
+cygnus-common/.project
+cygnus-ngsi/.classpath
+cygnus-ngsi/.project
+cygnus-twitter/.classpath
+cygnus-twitter/.project
+telefonica_checkstyle.xml
```

Comment:
telefonica_checkstyle.xml shouldn't be ignored, it's an artifact that we want to keep in repository

Can you elaborate on the rest of addition to this file, please? .gitignore should be keep to the minimum list and each new addition should be carefully analyzed.

Reply:
Sorry, .gitignore wasn't supposed to be part of the commit.  Our local telefonica_checkstyle.xml has chages to get it working with our development IDE. Was added to local gitignore to avoid commiting this unwanted changes.

This changes have been reverted in  7d08b4c